tests/lang/s05_pragmas/s02_misc/t03_hint_pragmas.nim

```diff
@@ -0,0 +1,42 @@
+discard """
+description: '''
+`{.hint.}` pragma test
+'''
+
+nimout: '''
+t03_hint_pragmas.nim(15, 7) Hint: User-provided hint message [User]
+t03_hint_pragmas.nim(19, 9) Hint: gen1 size of the argument is 8 [User]
+t03_hint_pragmas.nim(34, 6) Hint: 'declaredButNotUsed' is declared but not used [XDeclaredButNotUsed]
+'''
+
+
+"""
+
+{.hint: "User-provided hint message".}
+
+
+proc gen1[T](arg: T) =
+  {.hint: "gen1 size of the argument is " & $sizeof(arg).}
+
+gen1[int](1)
+
+when false:
```

Comment:
Can we make a const with a name indicating, skip because broken, and when on that, this way we know we're skipping it and it's searchable?